packages/contracts/contracts/default-apps/FreeBalanceApp.sol

```diff
@@ -5,22 +5,23 @@ import "../interfaces/CounterfactualApp.sol";
 import "../libs/LibOutcome.sol";
 
 
-contract ETHBucket is CounterfactualApp {
+contract FreeBalanceApp is CounterfactualApp {
```

Comment:
Should be called "LedgerApp" or similar since it now controls both the unassigned balance and the list of active apps

Reply:
I think FreeBalance is more specific than Ledger because there is no on-chain methods to withdraw from it like there is in Nitro, instead it represents general structural data on the StateChannel. Maybe something like `RootApp` or something like this is clearer but for now I think `FreeBalance` is OK